src/AElf.OS/Handlers/Network/BlockReceivedEventHandler.cs

```diff
@@ -11,7 +11,7 @@
 using Volo.Abp.DependencyInjection;
 using Volo.Abp.EventBus;
 
-namespace AElf.OS.Handlers
+namespace AElf.OS.Handlers.Network
```

Comment:
AElf.OS.Handlers.Network

what's the reference of the handler?

if it only uses network, we should put it under AElf.OS.Network, 
if it uses AElf.OS.Network and AElf.OS.xxxx, we should put it under AElf.OS

Reply:
AElf.OS.Handlers.Network is unacceptable